e2e/template.go

```diff
@@ -448,31 +448,39 @@ func baseTemplate(location string) *datamodel.NodeBootstrappingConfiguration {
 	}
 }
 
-func getNginxPodTemplate(nodeName string) string {
+func getHTTPServerTemplate(podName, nodeName string) string {
 	return fmt.Sprintf(`apiVersion: v1
 kind: Pod
 metadata:
-  name: %[1]s-nginx
+  name: %s
 spec:
   containers:
-  - name: nginx
-    image: mcr.microsoft.com/oss/nginx/nginx:1.21.6
+  - name: mariner
+    image: mcr.microsoft.com/cbl-mariner/busybox:2.0
     imagePullPolicy: IfNotPresent
-    readinessProbe:
+    command: ["sh", "-c"]
+    args:
+    - |
+      mkdir -p /www &&
+      echo '<!DOCTYPE html><html><head><title></title></head><body></body></html>' > /www/index.html &&
```

Comment:
doesn't nginx accomplish this already?

Reply:
nginx from mcr is a deprecated private image with a lot of stuff in it.

I wanted to use a minimal docker image to speed up pod start up. (it didn't improve much)

Reply:
I wanted to use something like alpine but couldn't find anything reasonable in MCR.

Not sure if we should use other container registries

Reply:
fair enough, I think this is fine as long as it works as we expect, just wanted to make sure